src/cmd-generate-hashlist

```diff
@@ -90,6 +90,8 @@ class HashListV1(dict):
             'ostree', 'checkout',
             '--repo=tmp/repo', '-U',
             self._metadata['ostree-commit'], checkout])
+        subprocess.check_call(['find', checkout, '-type', 'd', '-exec',
```

Comment:
I think the context from the git commit message would be useful to have in a comment here in the code.